src/System.Text.Http.Parser/HttpMethod.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace System.Text.Http.Parser
+{
+    public enum HttpMethod: byte
```

Comment:
It shipped in Kestreal 1.0. https://github.com/aspnet/KestrelHttpServer/blob/d3694f085a2df0c51286fdc4369bca8012d308fc/src/Microsoft.AspNetCore.Server.Kestrel/Internal/Http/HttpMethod.cs

@davidfowl, what's your compat bar for APIs in internal namespaces?